File: 백준/Bronze/4153. 직각삼각형/직각삼각형.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;
import java.util.StringTokenizer;


public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        StringTokenizer st;

        while (true) {
            int arr[] = new int[3];
            st = new StringTokenizer(br.readLine(), " ");
            for (int i = 0; i < arr.length; i++) {
                arr[i] = Integer.parseInt(st.nextToken());
            }

            if(arr[0] == 0 && arr[1] == 0 && arr[2] == 0)   break;
            Arrays.sort(arr);

            if (Math.pow(arr[0], 2) + Math.pow(arr[1], 2) == Math.pow(arr[2], 2)) {
                sb.append("right").append("\n");
            } else {
                sb.append("wrong").append("\n");
            }
        }
        System.out.println(sb);
    }
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
import sys

while True:
	arr = list(map(int, sys.stdin.readline().split()))
	arr.sort()
	if arr[0] == 0 and arr[1] == 0 and arr[2] == 0:
		break

	if arr[0] ** 2 + arr[1] ** 2 == arr[2] ** 2:
		print('right')
	else:
		print('wrong')